### PR TITLE
Gh 245 safe delivery

### PR DESCRIPTION
**Pull request by psugihara**
## Change description

Adds the ability to force delivery of an intercepted send.

UI polish:
- global 1px translate for active state on buttons/links
- fixed height bug in PreviewViewer causing window to scroll
 
test at http://localhost:3883/intercepts/mock

Screen Shot 2022-10-16 at 9 37 19 PM
Screen Shot 2022-10-16 at 9 38 35 PM
Screen Shot 2022-10-16 at 9 39 00 PM

Screen Shot 2022-10-16 at 9 39 19 PM
Screen Shot 2022-10-16 at 9 39 48 PM

Screen Shot 2022-10-16 at 9 44 36 PM

## Issue link

#245 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.

### CODE REVIEW

**Comment by alexfarrill** [comment]
Nice!

File: .eslintrc.json [hunk 1]
```diff
@@ -54,6 +54,12 @@
         "@typescript-eslint/no-empty-function": [
           "error",
           { "allow": ["arrowFunctions"] }
+        ],
+        "react/no-unknown-property": [
```
Comment (alexfarrill) [inline .eslintrc.json:58]: :tada

**Review by alexfarrill — CHANGES_REQUESTED** [review]
When I click "force deliver" I see 404 and in server:
```
mailing 404 /api/intercepts/undefined/forceDeliver 116ms
```

**Review by alexfarrill — APPROVED** [review]
works now!  thanks

do we need a more contrasting button color for the "force send" button? (red?) It is kind of hidden

**Comment by psugihara** [comment]
Yeah, good catch.

On design, I think it's correct for this to be subtle but legible (white on black is high contrast / legible). This is not something you will want to do very often but it will show every time you send and should not be a CTA. There are other designs in the Figma where it pops more, but I don't think that's the right hierarchy given the normal use-case for this page and dangerous consequences of clicking.  Monica thought a fade in would be nice and give a touch more prominence, I will try it.

**Comment by alexfarrill** [comment]
Sounds good 👌

**Pull request merged**